web/docs/usage/enrich-with-network-inventory/README.md

```diff
@@ -75,8 +75,10 @@ Let's use the `enrich` operator to add the subnet context to all IP address
 fields:
 
 ```
-from /tmp/sample.json
-| enrich subnets --field :ip
+load_file "/tmp/sample.json"
+read_json
+context::enrich "subnets", key=src_ip
+context::enrich "subnets", key=dest_ip
```

Comment:
Output needs to be regenerated here.

Reply:
That, and the example doesn't really make sense, as the second enrich overwrites the output from the previous one. The `into` option defaults to the context's name.

Reply:
I am going to resolve this but will add a comment in a related story to update this.

Reply:
The output still needs to be regenerated here right below.

Reply:
I.e., exchange this:

```
  "subnets": {
    "src_ip": {
      "value": "10.0.0.1",
      "timestamp": "2024-05-29T13:02:56.368882",
      "mode": "enrich",
      "context": {
        "subnet": "10.0.0.0/24",
        "owner": "Derek",
        "function": "servers"
      }
    },
    "dest_ip": {
      "value": "10.1.1.254",
      "timestamp": "2024-05-29T13:02:56.368882",
      "mode": "enrich",
      "context": null
    }
  }
```

with:

```
  "src_ip_context": {
    "subnet": "10.0.0.0/24",
    "owner": "Derek",
    "function": "servers"
  },
  "dest_ip_context": "context": null
```

Reply:
Okay, I manually replaced it.